### ISSUE TITLE
Possible improvements: more consistent double click in the navigator menu.

### ISSUE THREAD

**Issue by Mizar01** [dbeaver/dbeaver#16913]
I noted various non trivial behaviours of the double click in the db navigator.

For the following text 'Expected behaviour' means 'the way I would desire for it to work'

Double clicking on a group/folder name will open the right pane with all the db, but does not expand the group itself. I have to precisely pick the little triangle to expand the group. Expected default behaviour: expand the group.

Double clicking on a database will expand its structure. That's correct in my opinion. But it's in contrast with what happens for the sub structure folders.

Double click on a schema will open a right pane. Expected behaviour: expand the structure directly in the navigator.

... And so on. 

In short, any opening in the right pane should be done with a chosen right click action. Any double click should simply expand on the navigator itself, unless it is obvious that a right pane is necessary.

**Comment by uslss** [comment]
@Mizar01 this behavior could be changed in Preferences
![image](https://user-images.githubusercontent.com/31996417/176397697-e243c8a7-2b13-4a91-abb3-72354ba6d879.png)
